hack/patch-nix-hash.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+nix --log-format raw build ./dev/nix/
```

Comment:
Is installing nix part of the Makefile yet?

Reply:
No it isn't, don't really want to do that in the Makefile because the installation is not straightforward (see this: https://nixos.org/download). 


Do you think we should add it or is it okay to leave it out?